clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1570,4 +1577,189 @@ void handleFailure(Throwable throwable) {
         }, now);
         return new AlterConfigsResult(new HashMap<ConfigResource, KafkaFuture<Void>>(futures));
     }
+
+    /**
+     * Change the log directory for the specified replicas. This API is currently only useful if it is used
+     * before the replica has been created on the broker. It will support moving replicas that are already created after
+     * KIP-113 is fully implemented.
+     *
+     * Updates are not transactional so they may succeed for some resources while fail for others. The log directory for
```

Comment:
Sure. I removed the sentence `Updates are not transactional...` from this comment. Does this address the problem?